src/main/resources/view/CommandBox.fxml

```diff
@@ -1,9 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
+<?import java.net.URL?>
+<?import javafx.scene.control.Button?>
 <?import javafx.scene.control.TextField?>
 <?import javafx.scene.layout.StackPane?>
 
-<StackPane styleClass="stack-pane" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
-  <TextField fx:id="commandTextField" onAction="#handleCommandEntered" promptText="Enter command here..."/>
-</StackPane>
 
+<StackPane styleClass="stack-pane" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
```

Comment:
Remember to change from `xmlns="http://javafx.com/javafx/17"` to `xmlns="http://javafx.com/javafx/**11**"`